plugins/extractors/tableau/tableau.go

```diff
@@ -62,6 +63,7 @@ type Option func(*Extractor)
 // WithHTTPClient assign custom http client to the Extractor constructor
 func WithHTTPClient(hcl *http.Client) Option {
 	return func(e *Extractor) {
+		hcl.Transport = telemetry.NewHTTPTransport(hcl.Transport)
 		e.httpClient = hcl
 	}
```

Comment:
This is only used in tests. We should make changes in `plugins/extractors/tableau/client.go` (`NewClient`).

Reply:
done